tests/tests_api/test_computed_tables.py

```diff
@@ -0,0 +1,55 @@
+from kirby.api.ext import Topic
+from kirby.api import Table
+
+
+def process_generator(key):
+    def process_orders(orders):
+        return orders[key]
+
+    return process_orders
+
+
+def test_it_compute_table(kirby_app):
```

Comment:
This sounds like an integration test already, I'm not able to figure out what's going to come out of creating a table and adding a listener, and ... 

Probably a good idea to narrow down how each component works individually before running this one test, so I have multiple sources to understand what does what

Reply:
The tests are now splitted